src/net.h

```diff
@@ -550,13 +551,14 @@ class CNode
      * @param[in]   msg_bytes   The raw data
      * @param[out]  complete    Set True if at least one message has been
      *                          deserialized and is ready to be processed
-     * @param[out]  mapBytesPerMsg   A map of message types and the
-     *                               number of bytes received per type
+     * @param[out]  msgtype_countbytes   The message types mapped to a tuple
+     *                          that contains the total message count and
+     *                          total number of message bytes
      * @return  True if the peer should stay connected,
      *          False if the peer should be disconnected from.
      */
     bool ReceiveMsgBytes(Span<const uint8_t> msg_bytes, bool& complete,
-                         mapMsgTypeSize& msgtype_bytes) EXCLUSIVE_LOCKS_REQUIRED(!cs_vRecv);
+                         std::map<std::string, std::tuple<int, uint64_t>>& msgtype_bytes) EXCLUSIVE_LOCKS_REQUIRED(!cs_vRecv);
```

Comment:
Since this variable was changed to a `pair`, I prefer the name to reflect the two items in the pair (`msgtype` and `bytes`). But, per one of your earlier comments, if this gets changed to a `tie`, I think calling it a more generic `msg_stats` is the way to go, especially if a `tie` lets you access the elements by names that you define.

Reply:
No longer relevant since I reverted the change to add this new parameter to `ReceiveMsgBytes()`. Closing.